modules/PheWAS.py

```diff
@@ -232,7 +217,6 @@ def main(genesSet, inputFile, consequencesFile, diseaseMapping, skipMapping):
     parser.add_argument('-i', '--inputFile', required=True, type=str, help='Input .csv file with the table containing association details.')
     parser.add_argument('-c', '--consequencesFile', required=True, type=str, help='Input look-up table containing the variation consequences coming from the Variant Index.')
     parser.add_argument('-d', '--diseaseMapping', required=False, type=str, help='Input look-up table containing the phenotype mappings to an EFO ID.')
-    parser.add_argument('-g', '--genesSet', required=False, type=str, help='URL for the complete HGNC approved dataset in JSON format.')
     parser.add_argument('-o', '--outputFile', required=True, type=str, help='Name of the compressed json.gz output file containing the evidence strings.')
     parser.add_argument('-s', '--skipMapping', required=False, action='store_true', help='State whether to skip the disease to EFO mapping step.')
```

Comment:
This is quite marginally related: if we want to skip the disease mapping we can just simply not pass the diseaseMaping file. The presence or absence can be used as a flag to indicate weather we want or not want disease mapping. But this is not a blocker for sure.

Reply:
Thanks for the comment. This was implemented when the plan was to modularise the disease mapping process and maybe move it to the ETL.

I'll bear this comment in mind and will change all the parsers accordingly.